test/users/npadmana/blas/c2chapel.py

```diff
@@ -0,0 +1,99 @@
+""" A C to Chapel parser to help automate writing bindings.
```

Comment:
Also, given it's generality, it could and probably should be contributed in a separate shorter PR altogether.